### ISSUE TITLE
Trying to generate JSON-LD encoded EARL report using Webdriverjs

### ISSUE THREAD

**Issue by adilsofficial** [dequelabs/axe-core-npm#88]
I'm trying to generate the earl format result using axe earl reporter with webdriverjs, but unable to do that as there is no API documentation for this reporter. 
<code> var earlResults = reporter.createEarlReport(results);
        console.log(earlResults);</code>
Is there another way to call the reporter function to generate the earl reports?

**Comment by wonderfulspam** [comment]
Any news on this? I have a similar use case: I'm trying to combine `@axe-core/puppeteer` and `@axe-core/reporterEarl` and not having much luck.

```js
await page.goto(url);
await new AxePuppeteer(page).options( // Changing .options() to .configure() makes no difference
    {
        reporter: "rawEnv",
    }
).analyze(function(err, res) {
    console.log(err); // prints "null"
    console.log(res); // prints "undefined"
    console.log(createEarlReport(res)); // Fails because res is undefined
});
await page.close();
```

I'm not sure whether this is simply impossible or if I'm going about it the wrong way. Would it be possible to provide a self-contained example of integrating `@axe-core/reporterEarl` in one of the other `axe-core-npm` packages?
